stories/components/contact-us/contact-us.html.twig

```diff
@@ -1,4 +1,4 @@
-<section id="contact" class="bix-contact bix-section bix-section--primary">
+<section id="contact" class="bix-contact bix-section bix-section--primary" aria-label="Contact us">
```

Comment:
Added an aria label here to make this a landmark. Previously, axe devtools was complaining about this not being inside of a landmark section.